projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -989,6 +989,11 @@ public function update_data( $request ) {
 
 						$updated = (string) get_option( $option ) !== (string) $sub_value ? update_option( $option, $sub_value ) : true;
 					break;
+				case 'jetpack_subscriptions_from_name':
+					// If option value was the same, consider it done.
+					$sub_value = sanitize_text_field( $value );
```

Comment:
Might be good to hit with `trim()` here too?

Reply:
Trim is already done in update_option for this case.